contrib/nightly/plugins.yml

```diff
@@ -6,3 +6,11 @@
   description: |
     Tutor plugin to enable execution of untrusted code in secure sandboxes.
     In particular, this plugin is required to run Python-evaluated input XBlocks.
+- name: pod-autoscaling
+  src: -e git+https://github.com/eduNEXT/tutor-contrib-pod-autoscaling@v15.0.0#egg=tutor-contrib-pod-autoscaling
```

Comment:
Are you sure you want to install from v15.0.0 tag, and not the nightly branch?

Reply:
@regisb I just have a question here, `nightly` will always follow `main`? Is it like the latest version? According to [this](https://docs.tutor.overhang.io/tutorials/nightly.html), that's the case, I just wanted to make sure I'm not misunderstanding the purpose of the nightly branch

Reply:
@regisb fixed